docs/ref/models/querysets.txt

```diff
@@ -328,6 +328,13 @@ identical to::
         # Join
         Entry.objects.order_by('blog__id')
 
+.. versionadded:: 1.8
```

Comment:
I see the previous example is bad, but I'm going to fix that and this to make these annotations "self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.